src/Microsoft.ML.Data/Transforms/NormalizeColumn.cs

```diff
@@ -935,14 +936,14 @@ internal static partial class MinMaxUtils
                 Contracts.AssertValue(host);
                 host.AssertValue(args);
 
-                return CreateBuilder(new Normalizer.MinMaxColumn(
+                return CreateBuilder(new NormalizerEstimator.MinMaxColumn(
```

Comment:
>NormalizerEstimator [](start = 41, length = 19)

shall this be called NormalizingEstimator? Or ColumnNormalizingEstimator?
cc @Zruty0, @tomfinley

Reply:
NormalizingEstimator would be my preference